api/api/jobs/tenant_migrated_jobs.py

```diff
@@ -13,3 +15,12 @@ async def handle_tenant_migrated(event: TenantMigratedEvent, customer_service: C
 
 
 JOBS = [handle_tenant_migrated]
+
+
+@broker.task(retry_on_error=True)
+async def send_daily_digest(event: TenantMigratedEvent, customer_service: CustomerServiceDep):
+    await customer_service.send_daily_user_digest()
+
+
+# Will run 2 hours after signup
+SCHEDULED_JOBS = [(send_daily_digest, timedelta(hours=2))]
```

Comment:
maybe renamed to `DELAYED_JOBS`

It might be clearer, if you have a single jobs array with a NamedTuple `WithDelay`

So it would look like `JOBS = [handled_tenant_migrated, WithDelay(send_daily_digest, timedelta(hours=2))]` and handle the is instance in the job handler

Reply:
updated here https://github.com/WorkflowAI/WorkflowAI/pull/161/commits/e6ba35df22b5b5e6e0bd7a13afc28a28f622c3af